fl4health/clients/basic_client.py

```diff
@@ -79,6 +81,7 @@ def __init__(
         self.num_train_samples: int
         self.num_val_samples: int
         self.learning_rate: Optional[float] = None
+        self.warmed_up_module = warmed_up_module
```

Comment:
Super minor optional nitpick, but I think it's better organized if this line goes before the line above just to keep all the param setting lines together and the other variables that are not init params at the end.

Reply:
Sorry, I overlooked how big this init is in the first place. What I meant by the comment is to put this statement below line 57. But again, minor nitpick.

Reply:
That makes sense.